### PR TITLE
simple csv extractor example

### PR DESCRIPTION
**Pull request by msolvik**
For the CDF Bootcamp we need a simple csv extractor example that could be deployed as a Cognite Function.

This PR contains a simplified version of the existing csv-extractor sample that reads CSV files and uploads the content to a configurable database and table in CDF RAW. This extractor can be deployed as a Cognite Function.  This simple version differ from the csv-extractor sample by:
1. No state store
2. No Prometheus metrics
3. Added extraction pipeline in config
4. The config file path can be passed into extractor.main() so it can be retrieved from the data JSON when running as a Cognite Function

### CODE REVIEW

File: csv-extractor-simple/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+flake8 = "^3.9.2"
+mypy = "^0.910"
+black = "^20.8b1"
+
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry.scripts]
+csv_extractor = "csv_extractor.extractor:main"
```
Comment (timurgen) [inline csv-extractor-simple/pyproject.toml]: this one needs to be fixed in order to run it through poetry

**Review by timurgen — COMMENTED** [review]
Looks good as an example, I couldn't run it as typed-ast 1.4.* fails to build on Windows platform

**Review by timurgen — APPROVED** [review]

File: csv-extractor-simple/pyproject.toml (hunk 1 above)
Comment (msolvik) [reply csv-extractor-simple/pyproject.toml]: Thanks for pointing that out. I ran it with poetry shell and forgot to test this!

**Review by SunilVenekarKrishnamoorthy — APPROVED** [review]
Reviewed by Timur.

**Pull request merged**